src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPackageInstaller.cs

```diff
@@ -24,7 +24,8 @@
 namespace NuGet.VisualStudio
 {
     [Export(typeof(IVsPackageInstaller))]
-    public class VsPackageInstaller : IVsPackageInstaller
+    [Export(typeof(IVsPackageInstaller2))]
```

Comment:
Spoke offline to Justin about this. Exporting both allows all configurations:

1. Old caller can still get `IVsPackageInstaller`.
1. New caller can still get `IVsPackageInstaller` try to cast up with `as`.
1. New caller can get `IVsPackageInstaller2` to require the new functionality.